doc/reference/misc.md

```diff
@@ -1396,3 +1422,47 @@ Please document me!
 :::
 
 Please document me!
+
+
+
+## Func
```

Comment:
I would call this functional utilities, we'll add up to it in the future probably.